components/card/card.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div
+    :class="classes"
+    ref=card>
```

Comment:
1. If need ref the correct declare is `ref="card"`
2. `ref` seem like is useless.